perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2155,28 +2155,23 @@ private boolean isMemberInGracePeriod(Map<String, String> membershipExpirationRu
 
 		int amount = Integer.valueOf(m.group(1));
 
-		int field;
+		TemporalUnit field;
```

Comment:
Maybe I would use a different name for this variable. Maybe `gracePeriodTimeUnit` ?

Reply:
Okay, that is more accurate, thanks.